_posts/2018-06-29-14-04-netcore-7-0.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+We're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
+ASP.NET Web API OData 7.0 is available with two new packages:
```

Comment:
>new  [](start = 48, length = 4)

Should we use "new" here? the .NETCore is certainly new; the other is a (significant) new release of the 6.x. I'm afraid that "new" makes it sound like there are now 3 options; what they are currently using, 7.0 core, and 7.0 classic.  Maybe change to "ASP.NET WebAPI OData 7.0 is available in two packages:"

Reply:
+1

Reply:
#resolved.